CODEOWNERS

```diff
@@ -160,6 +160,7 @@ build.json @home-assistant/supervisor
 /tests/components/bond/ @bdraco @prystupa @joshs85 @marciogranzotto
 /homeassistant/components/bosch_shc/ @tschamm
 /tests/components/bosch_shc/ @tschamm
+/homeassistant/components/bouncie/ @mandarons
```

Comment:
This wasn't applied

Reply:
Yeah, `pre-commit` hook removed these changes here: https://github.com/home-assistant/core/commit/f8787706e6f99a081a08047aab9009405171ff7e and only then, let me commit my code. Let's see if it sticks this time.

Reply:
OK, so that's odd:
hassfest (`python3 -m script.hassfest`) is failing too on the pipeline, it's either because of:
- this line that it conciders not to be here,
- OR the missing lib addition into `requirements_test_all.txt`, even if we don't use it on our tests

Reply:
Added to `requirements_test_all.txt`.

Reply:
Thanks.

Emmmm, since the pipeline isn't passing again, just launch: `python3 -m script.gen_requirements_all`, and `python3 -m script.hassfest`, and we will keep the result as it is 💁‍♂️

Reply:
This is missing again

Reply:
@epenet : Yeah, `pre-commit` check removed those. See above conversation. Am I missing anything?

Reply:
I think the problem is that you are missing `__init__.py` in tests folder.